utils/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+#[macro_use]
+extern crate vmm_sys_util;
+use std::os::raw::c_int;
+pub use vmm_sys_util::{errno, eventfd, fam, ioctl, signal, tempdir, tempfile, terminal};
```

Comment:
It looks like errno, fam, ioctl, tempfile and tempdir are not yet used in the Firecracker code. Should we re-export them in this case?

Reply:
You're right, we can remove fam, ioctl, tempfile and tempdir, but errno is still used here: [019c3e7](https://github.com/firecracker-microvm/firecracker/pull/1413/commits/019c3e795f9d43f52598e12fe16f455ee20f6a1c)